### PR TITLE
Call handleMessage in a goroutine so it prevents possible UDP packet loss

### PR DESCRIPTION
**Pull request by marcosnils**
Solution to problem described here: 

https://groups.google.com/forum/#!topic/influxdb/HEQbcuz1NbY

### CODE REVIEW

File: collectd/collectd.go [hunk 1]
```diff
@@ -107,7 +107,8 @@ func (s *Server) serve() {
 			continue
 		}
 		if n > 0 {
-			s.handleMessage(buffer[:n])
+			//Make packaet processing in a separate goroutine so it prevents UDP packet loss
```
Comment (otoolep) [inline collectd/collectd.go]: Nit: Needs a whitespace between `//` and `Make`.

**Comment by otoolep** [comment]
Build script has been fixed, please rebase so CI testing can re-run.

**Comment by marcosnils** [comment]
@otoolep rebased and comments fixed!.

**Comment by marcosnils** [comment]
@otoolep tests are still failing....

**Pull request closed without merging**